src/rdpc/index.ts

```diff
@@ -82,20 +102,15 @@ export const indexRdpcData = async (
 
   logger.info(`Begin bulk indexing donors of program ${programId}...`);
 
-  await esClient.bulk(
-    {
-      body: toEsBulkIndexActions<EsDonorDocument>(
-        targetIndexName,
-        (donor) => preExistingDonorHits[donor.donorId]?._id
-      )(esDocuments),
-      refresh: "true",
-    },
-    (error, response) => {
-      if (error) {
-        logger.error(response);
-      }
-    }
```

Comment:
this changes the return signature of `esClient.bulk` to be a callback-based rather than promise-based async operation, so needs to be removed for `await` to actually work